WowPacketParser/WowPacketParser.csproj

```diff
@@ -91,6 +91,7 @@
     <LangVersion>6</LangVersion>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include=”netstandard” />

```

Comment:
You will not make any build system happy with these weird language specific quote marks

Reply:
already noticed it xD